package/MDAnalysis/analysis/base.py

```diff
@@ -90,13 +90,76 @@
    plt.ylabel("RMSD (Å)")
 
 
+Using parallelization in your analysis
```

Comment:
Is this section aimed at users writing new analyses or people running in-built analyses? If it's the latter, the[ analysis modules docs](https://github.com/MDAnalysis/mdanalysis/blob/develop/package/doc/sphinx/source/documentation_pages/analysis_modules.rst) might be a better place for it.